src/utils/data/animals.mock.json

```diff
@@ -1,89 +1,71 @@
 {
   "edges": [
+    { "source": "9d989e8d", "target": "f42dab68" },
+    { "source": "85c4cf64", "target": "f42dab68" },
+    { "source": "0ae9e4de", "target": "f42dab68" },
+    { "source": "09f5edeb", "target": "f42dab68" },
+    { "source": "f42dab68", "target": "2cd4ba93" },
+    { "source": "f42dab68", "target": "fac8f1a3" },
+    { "source": "e4951252", "target": "443cf06a" },
+    { "source": "c38d4c6a", "target": "443cf06a" },
+    { "source": "f1f1425b", "target": "443cf06a" },
+    { "source": "9d989e8d", "target": "443cf06a" },
+    { "source": "443cf06a", "target": "6525f2e6" },
+    { "source": "443cf06a", "target": "091b5035" },
+    { "source": "735bde08", "target": "1c18dc89" },
+    { "source": "1c18dc89", "target": "5f7e0e20" },
+    { "source": "a761759c", "target": "c8c182ec" },
+    { "source": "24e06541", "target": "c8c182ec" },
+    { "source": "c8c182ec", "target": "0049a504" }
+  ],
+  "layers": ["raw", "feature", "model output", "primary"],
+  "modular_pipelines": [
     {
-      "source": "e4951252",
-      "target": "443cf06a"
-    },
-    {
-      "source": "c38d4c6a",
-      "target": "443cf06a"
-    },
-    {
-      "source": "f1f1425b",
-      "target": "443cf06a"
-    },
-    {
-      "source": "9d989e8d",
-      "target": "443cf06a"
-    },
-    {
-      "source": "443cf06a",
-      "target": "6525f2e6"
-    },
-    {
-      "source": "443cf06a",
-      "target": "091b5035"
-    },
-    {
-      "source": "9d989e8d",
-      "target": "f42dab68"
-    },
-    {
-      "source": "85c4cf64",
-      "target": "f42dab68"
-    },
-    {
-      "source": "0ae9e4de",
-      "target": "f42dab68"
-    },
-    {
-      "source": "09f5edeb",
-      "target": "f42dab68"
-    },
-    {
-      "source": "f42dab68",
-      "target": "2cd4ba93"
+      "id": "pipeline1",
+      "name": "Pipeline1"
     },
     {
-      "source": "f42dab68",
-      "target": "fac8f1a3"
+      "id": "pipeline1.data_engineering",
+      "name": "Data Engineering"
     },
     {
-      "source": "2cd4ba93",
-      "target": "e27376a9"
+      "id": "pipeline1.data_science",
+      "name": "Data Science"
     },
     {
-      "source": "6525f2e6",
-      "target": "e27376a9"
+      "id": "pipeline2",
+      "name": "Pipeline2"
     },
     {
-      "source": "e27376a9",
-      "target": "1769e230"
+      "id": "pipeline2.data_science",
+      "name": "Data Science"
```

Comment:
@limdauto Added `pipeline2.data_science` and `pipeline1.data_science` to show the "nameclash" case.